src/main/kotlin/uk/gov/gdx/datashare/services/ScheduledJobService.kt

```diff
@@ -23,4 +28,14 @@ class ScheduledJobService(
     val unconsumedEventsCount = acquirerEventRepository.countByDeletedAtIsNull()
     gauge!!.set(unconsumedEventsCount)
   }
+
+  @Scheduled(fixedRate = 1, timeUnit = TimeUnit.MINUTES)
+  @SchedulerLock(name = "deleteConsumedGroSupplierEvents", lockAtMostFor = "45s", lockAtLeastFor = "15s")
```

Comment:
The `lockAtMostFor` should be a bit more than the time limit. It might take ~ 1 second to build the list of events and finish up the last one to be processed. I'd set this to 60s